### PR TITLE
[doc] where statement updated

### PR DESCRIPTION
**Pull request by wilsonyou**

### CODE REVIEW

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md [hunk 1]
```diff
@@ -13,13 +13,40 @@ WHERE <expression> [ AND | OR <expression> ...])
...
-nebula> GO FROM 201 OVER e1 WHERE e1.prop1 >= 17
--- 起点 v1 的 prop1 属性与终点 v2 的 prop2 属性值相等
-nebula> GO FROM 201 OVER e1 WHERE $^.v1.prop1 == $$.v2.prop2
--- 多种逻辑组合
-nebula> GO FROM 201 OVER e1 WHERE ((e3.prop3 < 0.5) \
-   OR ($^.v4.prop4 != "hello")) AND $$.v5.prop5 == "world"
---下面这个条件总是为 TRUE
-nebula> GO FROM 201 OVER e1 WHERE 1 == 1 OR TRUE
+-- 边 follow 的 degree 属性大于 90。
+nebula> GO FROM 100 OVER follow WHERE follow.degree>90;
```
Comment (whitewum) [inline docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md]: space between >

Comment (whitewum) [reply docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md]: And please check lint by yourself

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md [hunk 2]
```diff
@@ -8,18 +8,45 @@ WHERE <expression> [ AND | OR <expression> ...])
...
-nebula> GO FROM 201 OVER e1 WHERE e1.prop1 >= 17
--- the source vertex v1's property prop1 is equivalent with dest vertex v2's property prop2
-nebula> GO FROM 201 OVER e1 WHERE $^.v1.prop1 == $$.v2.prop2
--- logical combination is allowed
-nebula> GO FROM 201 OVER e1 WHERE ((e3.prop3 < 0.5) \
-   OR ($^.v4.prop4 != "hello")) AND $$.v5.prop5 == "world"
--- always TRUE
-nebula> GO FROM 201 OVER e1 WHERE 1 == 1 OR TRUE
+-- the degree property of edge follow is greater than 90.
+nebula> GO FROM 100 OVER follow WHERE follow.degree>90;
```
Comment (Amber1990Zhang) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md]: space here

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md [hunk 3]
```diff
@@ -8,18 +8,45 @@ WHERE <expression> [ AND | OR <expression> ...])
...
+-- the following result is returned:
+===============
+| follow._dst |
+===============
+| 101         |
+---------------
+-- the property value of source vertex, player 104, is equal to the property value of dest vertex, player 103.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md]: Find the dest vertex that share the samge age with the source vertex

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md [hunk 4]
```diff
@@ -8,18 +8,45 @@ WHERE <expression> [ AND | OR <expression> ...])
...
+nebula> GO FROM 100 OVER follow WHERE follow.degree > 90 OR $$.player.age != 33 AND $$.player.name != "Tony Parker";
+-- the following result is returned:
+===============
+| follow._dst |
+===============
+| 101         |
+---------------
+| 106         |
+---------------
+-- always TRUE.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md]: what always true? please in detail

**Pull request merged**